snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/SnomedCompositeImporter.java

```diff
@@ -410,13 +408,14 @@ protected void createSnomedVersionFor(final String lastUnitEffectiveTimeKey) {
 				final Date effectiveDate = EffectiveTimes.parse(lastUnitEffectiveTimeKey, DateFormats.SHORT);
 				final String formattedEffectiveDate = EffectiveTimes.format(effectiveDate);
 				
-				final ITagConfiguration configuration = TagConfigurationBuilder.createForRepositoryUuid(SnomedDatastoreActivator.REPOSITORY_UUID, formattedEffectiveDate)
-					.setBranchPath(snomedBranchPath)
-					.setUserId(importContext.getUserId())
-					.setParentContextDescription(DatastoreLockContextDescriptions.IMPORT)
-					.build();
-				
-				ApplicationContext.getInstance().getService(ITagService.class).tag(configuration);
+				RepositoryRequests
```

Comment:
This part should create a code system version as well.

Reply:
The previous version of the code did not create them at this part. The importer commits the new versions at the end of the import.